Formula/dar.rb

```diff
@@ -10,6 +10,19 @@ class Dar < Formula
     sha256 "a74696553c86f94dfd1ebdad29ad9c5d22c79d78fc023e43bb86f272fbfc4d87" => :mavericks
   end
 
+  devel do
+    url "https://downloads.sourceforge.net/project/dar/dar/2.5.5/dar-2.5.5.tar.gz"
+    sha256 "aa77d6778206b00ad7b37d5f615aafff3ea770b35d96625429c14ff67b78466d"
+
+    depends_on "gcc" if MacOS.version <= :el_capitan
```

Comment:
As far as I know that a legitimate linkage issue as well, because things compiled against the system link to `/usr/lib/libstdc++.6.dylib` for example.